azdev/operations/testtool/profile_context.py

```diff
@@ -28,15 +28,17 @@ def __enter__(self):
         if self.target_profile is None or self.target_profile == self.origin_profile:
             display('The tests are set to run against current profile "{}"'.format(self.origin_profile))
         else:
-            result = cmd('az cloud update --profile {}'.format(self.target_profile),
-                         'Switching to target profile "{}"...'.format(self.target_profile))
-            if result.exit_code != 0:
-                raise CLIError(result.error.output.decode('utf-8'))
+            display('Switching to target profile "{}"...'.format(self.target_profile))
+            cli = get_default_cli()
+            cli.invoke(["cloud", "update", "--profile", self.target_profile], out_file=open(os.devnull, 'w'))
```

Comment:
I don't feel the necessity of replacing a subprocess call with `cli.invoke`.

Reply:
to avoid run az in windows